src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4133,6 +4146,13 @@ private bool HasTopLevelNullabilityConversion(TypeWithAnnotations source, TypeWi
                 _conversions.HasTopLevelNullabilityImplicitConversion(source, destination);
         }
 
+        /// <summary>
+        /// Gets the conversion node for passing to <see cref="VisitConversion(BoundConversion, BoundExpression, Conversion, TypeWithAnnotations, TypeWithState, bool, bool, bool, AssignmentKind, ParameterSymbol, bool, bool, bool, Optional{LocalState}, bool, Location)"/>,
+        /// if one should be passed.
+        /// </summary>
+        private static BoundConversion GetConversionOperandIfApplicable(BoundExpression conversionOpt, BoundExpression convertedNode) =>
```

Comment:
>GetConversionOperandIfApplicable [](start = 39, length = 32)

Would it be useful to assert that the parameters are related via `RemoveConversion`?